project/frontend/src/index.js

```diff
@@ -10,18 +10,26 @@ import Header from './pages/Header.js';
 import HomePage from './pages/HomePage.js';
 import Footer from './pages/Footer.js';
 
+import RestaurantCard from './components/RestaurantCard.js';
+
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
     <Authentication>
       <Router>
         <Switch>
+          
+          <Route path='/test'>
```

Comment:
is this the line that gets the card to show up on the /test URL ?

Reply:
Yeah, whatever children are added under a `Route` component will be the ones visible on that page. It is kind a false page though, when you enter a new url the page doesn't actually reload it just edits the DOM. So when you go from '/' to '/test', React replaces the children under `<Route path='/'>` with <RestaurantCard />.